cmd/img_mgmt/port/zephyr/src/zephyr_img_mgmt.c

```
@@ -214,7 +214,7 @@ img_mgmt_impl_write_image_data(unsigned int offset, const void *data,
 		}
 	}
 
-	if (offset != ctx->bytes_written + ctx->buf_bytes) {
+	if (offset != ctx->fbw->bytes_written + ctx->fbw->buf_bytes) {
```

You don't need to `#include <storage/fbw.h>` for this to work?

---

This is included in the `flash_img.h` file, and these are only members of the `flash_img_ctx` struct. So it should be fine. CI will fail hard if something is missing :)

---

> CI will fail hard if something is missing :)

But what CI? :rof:

---

The zephyr CI builds these files, and it was a failure there that pointed me here :)